notebooks/code_sharing/capital_markets/quickstart_option_pricing_models_quantlib.ipynb

```diff
@@ -0,0 +1,1088 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9b8bd96",
+   "metadata": {},
+   "source": [
+    "<a id='toc2_'></a>\n",
+    "\n",
+    "## Install the client library\n",
+    "\n",
+    "The client library provides Python support for the ValidMind Developer Framework. To install it:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c9cbaa5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -q validmind"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "428ec832",
+   "metadata": {},
+   "source": [
+    "<a id='toc3_'></a>\n",
+    "\n",
+    "## Initialize the client library\n",
+    "\n",
+    "ValidMind generates a unique _code snippet_ for each registered model to connect with your developer environment. You initialize the client library with this code snippet, which ensures that your documentation and tests are uploaded to the correct model when you run the notebook.\n",
+    "\n",
+    "<a id='toc3_1_'></a>\n",
+    "\n",
+    "### Get your code snippet\n",
+    "\n",
+    "1. In a browser, [log in to ValidMind](https://docs.validmind.ai/guide/configuration/log-in-to-validmind.html).\n",
+    "\n",
+    "2. In the left sidebar, navigate to **Model Inventory** and click **+ Register new model**.\n",
+    "\n",
+    "3. Enter the model details and click **Continue**. ([Need more help?](https://docs.validmind.ai/guide/model-inventory/register-models-in-inventory.html))\n",
+    "\n",
+    "   For example, to register a model for use with this notebook, select:\n",
+    "\n",
+    "   - Documentation template: `Capital markets`\n",
+    "\n",
+    "   You can fill in other options according to your preference.\n",
+    "\n",
+    "4. Go to **Getting Started** and click **Copy snippet to clipboard**.\n",
+    "\n",
+    "Next, replace this placeholder with your own code snippet:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f8e48dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Replace with your code snippet\n",
+    "import validmind as vm\n",
+    "\n",
+    "vm.init(\n",
+    "  api_host = \"...\",\n",
+    "  api_key = \"...\",\n",
+    "  api_secret = \"...\",\n",
+    "  model = \"...\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee58640b",
+   "metadata": {},
+   "source": [
+    "<a id='toc4_'></a>\n",
+    "\n",
+    "## Initialize the Python environment\n",
+    "\n",
+    "Next, let's import the necessary libraries and set up your Python environment for data analysis:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "86a8da01",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from scipy.optimize import minimize\n",
+    "import yfinan\n",
+    "importib as ql\n",
```

Comment:
Typo here 😬 

```
import quantlib as ql
```

Reply:
Actually, still get an error:

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[5], [line 7](vscode-notebook-cell:?execution_count=5&line=7)
      [5](vscode-notebook-cell:?execution_count=5&line=5) from scipy.optimize import minimize
      [6](vscode-notebook-cell:?execution_count=5&line=6) import yfinance as yf
----> [7](vscode-notebook-cell:?execution_count=5&line=7) import quantlib as ql
      [8](vscode-notebook-cell:?execution_count=5&line=8) from validmind.tests import run_test

ModuleNotFoundError: No module named 'quantlib'
```

Reply:
it's  `QuantLib`. Updated the notebook now.